index.html

```diff
@@ -12,25 +12,30 @@
 </head>
 <body>
     <div class="menu">
-        <div class="bigsun"><img src="icons/bigsun.png" alt=""></div>
-        <h1>Your forecast app</h1>
-        <div class="search">
-            <span class="enter">enter your city
-                <div class="search__icon"><img src="icons/search.png" alt=""></div>
-            </span>
+        <div class="block__menu">
+            <div class="bigsun"><img src="icons/bigsun.png" alt=""></div>
+            <h1 class="title__one">Your forecast app</h1>
+            <div>
+                <div>
+                    <form name="search" method="post">
+                        <input class="enter" type="text" value="enter your city">
+                        <div class="search__icon"><a href="#"><img src="icons/search.png" alt=""></a></div>
+                    </form>
+                </div>
+            </div>
+            <h3 class="title__three"><a href="#">give me a random one</a></h3>
         </div>
-        <h3>give me a random one</h3>
     </div>
     <div class="london">
+        <div class="back"></div>
+        <div class="back__search"></div>
         <div class="prognoz">
-            <div class="back"></div>
-            <div class="back__search"></div>
             <div class="weather__picture--chilly">
-                <span class="temperature">10</span>
+                <div class="temperature">10</div>
                 <div class="weather__small"><img src="icons/smallsun.png" alt=""></div>
             </div>
-            <span class="title">London</span>
-            <span class="sub__title">11. 06. 2023</span>
+            <div class="title">London</div>
+            <div class="sub__title">11. 06. 2023</div>
             <div class="week">
                 <div class="day">
                     <div class="calender calender--one"><img src="icons/calender.png" alt=""></div>
```

Comment:
1. Replace `calender ` with `calendar `
2. Remove `calender--one` and similar classes. In the future, we'll generate data from JS, and using content in CSS will complicate the task